components/CompletionScoreCard.tsx

```diff
@@ -0,0 +1,213 @@
+          justifyContent: 'space-between',
+          alignItems: 'center',
+          marginBottom: '1em',
+        }}
+      >
+        <Box>
+          <Typography variant="h5">{data.title}</Typography>
+          <Box
+            component="span"
+            sx={{
+              display: 'inline-flex',
+              alignItems: 'center',
+              textDecoration: 'none',
+              '&:hover': {
+                borderBottom: '1px solid',
+              },
+            }}
+          >
+            <Link
+              href={data.linkHref}
+              variant="body2"
+              sx={{
+                display: 'inline-flex',
+                alignItems: 'center',
+                textDecoration: 'none',
+              }}
+            >
+              {data.linkText}
+              <ArrowUpwardIcon
+                sx={{
+                  fontSize: '1.1em',
+                  transform: 'rotate(45deg)',
+                  marginLeft: '0.3em',
+                }}
+              />
+            </Link>
+          </Box>
+        </Box>
+        <Button
+          sx={{
+            backgroundColor: 'transparent',
+            color: 'primary.main',
+            '&:hover': {
+              backgroundColor: 'transparent',
+              color: 'primary.main',
+              border: '1px solid',
+              borderColor: 'brand.800',
+            },
+          }}
+          variant="contained"
+          color="primary"
+          href="/dashboard"
+        >
+          Explore city dashboard{' '}
+          <ArrowUpwardIcon
+            sx={{
+              transform: 'rotate(45deg)',
+              fontSize: '1.2em',
+              marginLeft: '0.5em',
+            }}
+          />
+        </Button>
+      </Box>
+      <Box
+        sx={{
+          width: '60%',
+          backgroundColor: '#f0f0f0',
+          padding: '1em',
+          borderRadius: '0.5em',
+        }}
+      >
+        <Box
+          sx={{ display: 'flex', alignItems: 'center', marginBottom: '0.5em' }}
+        >
+          <FiberManualRecordIcon sx={{ color: 'red', marginRight: '0.5em' }} />
+          <Typography variant="subtitle1">
+            Data completion score: {completionPercentage.toFixed(0)}%
+          </Typography>
+        </Box>
+        <Typography variant="body1" paragraph>
+          Your score assesses the completeness of city data, assigning weights
+          based on the importance of city-specific values for precise
+          projections. Enter values in the Data Collection. Centre below to
+          boost your score and enable more accurate projections.
+        </Typography>
+        <Box sx={{ display: 'flex', gap: '0.5em' }}>
+          {scores.map((score) => (
+            <Chip
+              key={score.status}
+              icon={
+                <FiberManualRecordIcon
+                  sx={{
+                    color:
+                      score.status === 'High'
+                        ? 'red'
```

Comment:
We can use theme colours here for consistency `error.main`, `warning.main` and `success.main`. And if there are ever any properties that don't support [theme aware properties](https://mui.com/system/getting-started/the-sx-prop/#theme-aware-properties) we can always use the theme object like previously in styled components

e.g. 

```ts
          sx={{
            whatever: (theme) => theme.palette.warning.main,
          }}
```